projects/core/src/main/resources/data/computercraft/lua/rom/apis/textutils.lua

```diff
@@ -940,6 +940,32 @@ function urlEncode(str)
     return str
 end
 
+--- Splits a string in a table.
+--
+-- @tparam string string_to_split The string to split.
+-- @tparam string pattern At what pattern to split the string.
+-- @treturn table A table containing the splitted strings.
+-- @usage args = textutils.splitString("arg1 arg2", " ")
+-- @since edit this if it goes live (I hope it does)
+local function split(string_to_split, pattern)
+    local Table = {}
```

Comment:
Avoid having variable names that are the same as globals, even if a letter is capitalized. This can lead to mistakes down the line and overwrite the global value.